FIX - Folder structure MOSCOW statements (#402)
DILCISBoard/E-ARK-CSIP: specification/implementation/structure/index.md

# 4.	CSIP structure

> The preferred implementation of the logical model described in [Principle 3.6](#principle-36) is a strict physical (folder) structure precisely following the logical structure. While the specification doesn't prohibit alternative implementations of the conceptual model aren't prohibited by the CSIP it isn't recommended practise.

Reviewer: Remove duplication of "prohibit" from the last sentence:

```suggestion
The preferred implementation of the logical model described in [Principle 3.6](#principle-36) is a strict physical (folder) structure precisely following the logical structure. While the specification does not prohibit alternative implementations of the conceptual model, such practise is not recommended.
```
